source/install_scripted_online.rst

```diff
@@ -0,0 +1,75 @@
+Online Scripted Installation
+----------------------
+The scripted installation runs an installer that uses Ansible to install the command line tool, quipucords server image, and the database image. When you run the scripted installation, the server is installed and started as described in `Configuring and Starting Quipucords <install.html#config-and-start>`_. However, you can change some of the defaults used by the scripted installation with the `Installation Options <install.html#install-opts>`_.
+
+Obtaining the Installer
+^^^^^^^^^^^^^^^^^^^^^^^
+1. Download & Install the quipucords installer rpm.
+
+- RHEL & CentOS 6::
+
```

Comment:
Are we changing the name for the rpms here from `quipucords_installer` to `qpc_tools`?

Reply:
yep